control/autoware_vehicle_cmd_gate/src/vehicle_cmd_gate.hpp

```diff
@@ -183,6 +184,11 @@ class VehicleCmdGate : public rclcpp::Node
     this, "input/emergency/gear_cmd"};
   void onEmergencyCtrlCmd(Control::ConstSharedPtr msg);
 
+  // Previous Turn Indicators, Hazard Lights and Gear
+  TurnIndicatorsCommand::SharedPtr prev_turn_indicator;
+  HazardLightsCommand::SharedPtr prev_hazard_light;
+  GearCommand::SharedPtr prev_gear;
```

Comment:
As other variables, please add an underscore at the end of global variables.

Reply:
Sorry, I fixed this in [a0944bb](https://github.com/autowarefoundation/autoware.universe/pull/8084/commits/a0944bb870ada6b73727a1d4a0423990b2a6844b)